content/source/docs/cloud/integrations/service-now/service-catalog.html.md

```diff
@@ -0,0 +1,160 @@
+
+-> **ServiceNow Role:** `admin`
+
+First, add a Service Catalog for use with the Terraform integration. Depending
+on your organization's needs, you might use a single service catalog, or
+several. If you already have a Service Catalog to use with Terraform, skip to
+the next step.
+
+1. In ServiceNow, open the Service Catalog > Catalogs view by searching for
+   "Service Catalog" in the left-hand navigation.
+   1. Click the plus sign in the top right.
+   1. Select "Catalogs > Terraform Catalog > Title and Image" and choose a
+      location to add the Service Catalog.
+   1. Close the "Sections" dialog box by clicking the "x" in the upper right-hand
+      corner.
+
+-> **Note:** In step 1, be sure to choose "Catalogs", not "Catalog" from the
+left-hand navigation.
+
+## Configure VCS Repositories
+
+-> **ServiceNow Roles:** `admin` or `x_terraform.vcs_repositories_user`
+
+To make infrastructure available to your users, you must add one or more VCS
+repositories containing Terraform configuration to the Terraform service
+catalog.
+
+1. In ServiceNow, open the "Terraform > VCS Repositories" table by searching for
+   "terraform" in the left-hand navigation.
+1. Click "New" to add a VCS repository.
+    - Name: The name for this repository. This name will be visible to end
+      users, and does not have to be the same as the repository name as defined
+      by your VCS provider. Ideally it will succinctly describe the
+      infrastructure that will be provisioned by Terraform from the repository.
+    - OAuth Token ID: The OAuth Token ID that from your Terraform Cloud
+      organization's VCS providers settings. This ID specifies which VCS
+      provider configured in Terraform Cloud hosts the desired repository.
+    - Identifier: The VCS repository that contains the Terraform configuration
+      for this workspace template. Repository identifiers are determined by your
+      VCS provider; they typically use a format like
+      `<ORGANIZATION>/<REPO_NAME>` or `<PROJECT KEY>/<REPO_NAME>`. Azure DevOps
+      repositories use the format `<ORGANIZATION>/<PROJECT>/_git/<REPO_NAME>`.
+    - The remaining fields are optional.
+      - Branch: The branch within the repository, if different from the default
+        branch.
+      - Working Directory: The directory within the repository containing
+        Terraform configuration.
+      - Terraform Version: The version of Terraform to use. This will default to
+        the latest version of Terraform supported by your Terraform Cloud
+        instance.
+1. Click "Submit".
+
+![Screenshot: ServiceNow New VCS Repository](./images/service-now-vcs-repository.png "Screenshot of the ServiceNow Terraform New VCS Repository page")
+
+## Configure a Variable Set
+
+Most Terraform configuration can be customized with Terraform variables or
+environment variables. You can create a Variable Set within ServiceNow to
+contain the variables needed for a set of configuration. Your Terraform Admin
```

Comment:
```suggestion
contain the variables needed for a given configuration. Your Terraform Admin
```

Conversely, we use the singular noun for "the pile of .tf files that gets treated as a single workspace or working directory."